docs/docs/index.md

```diff
@@ -5,52 +5,54 @@ Welcome to zamba's documentation!
 <div class="embed-responsive embed-responsive-16by9" width=500>
     <iframe width=600 height=340 class="embed-responsive-item" src="https://s3.amazonaws.com/drivendata-public-assets/monkey-vid.mp4" frameborder="0" allowfullscreen=""></iframe>
 </div>
+<!-- TODO: what is the above supposed to do? it doesn't look like it's working><!-->
 
 *Zamba means "forest" in the Lingala language.*
 
 Zamba is a command-line tool built in Python to automatically identify the
-species seen in camera trap videos from sites in central Africa. Using the
+species seen in camera trap videos from sites in central and west Africa. Using the
 combined input of various deep learning models, the tool makes predictions
-for 23 common species in these videos (as well as blank, or, "no species
+for 31 common species in these videos (as well as blank, or, "no species
 present").
 
+*New in Zamba v2:* Zamba now has an additional model trained on 11 common European species. <!--TODO: add more details about where from><!-->
+
```

Comment:
Where was this data from? Add details on what region/countries in Europe if it's for a specific region

Reply:
You can look in the metadata CSVs in the `zamba-algorithms` project to find a bunch of this information.